guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+- `version`: Shows the implementation name and software version of the mint. Consider displaying this in the mint details to inform users about the mint's current software status.
+
+</div>
+
+#### Mint Descriptions
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/cashu-id2.png"
+   retina = "/assets/images/guide/how-it-works/ecash/cashu-id2@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/ecash/cashu-id2@2x.png"
+   alt-text = "Mint descriptions display in a wallet interface"
+   width = 250
+   height = 338
+   layout = "float-right-desktop -background -shadow"
+   caption = "Different mint operator may use the description fields differently. Consider this when designing your mint details screen."
+%}
+
+- `description`: A short description of the mint. Consider displaying this in mint listings or as a subtitle under the mint name to give users a quick overview.
+- `description_long`: A more detailed description. This can be displayed on a dedicated mint information page or in an expandable section for users who want more details.
+
+</div>
+
+#### Contact Information
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/cashu-id3.png"
+   retina = "/assets/images/guide/how-it-works/ecash/cashu-id3@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/ecash/cashu-id3@2x.png"
+   alt-text = ""
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+   caption = "Contact information is optional for the mint operator to include. If there is no contact information, consider not displaying this metadata."
+%}
+
+- `contact`: An list of contact methods for the mint operator. Consider displaying this in the mint details, using appropriate icons for each contact method (e.g., nostr, email icon, website icon) to make it easy for users to reach out if needed.
+
+</div>
+
+#### Important Announcements
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/cashu-id4.png"
+   retina = "/assets/images/guide/how-it-works/ecash/cashu-id4@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/ecash/cashu-id4@2x.png"
+   alt-text = "Example of important announcements display in a wallet interface"
+   width = 250
+   height = 390
+   layout = "float-right-desktop -background -shadow"
+   caption = "A message of the day can consists of anything the mint operator wants to tell the user. It can for example be used to announce a new feature or a upcoming maintenance."
+%}
+
+- `motd` (Message of the Day): Used for important announcements. Consider displaying this in a prominent, dismissible notification area when users interact with the mint, making sure they don't miss any critical information.
```

Comment:
There could also be a small notification icon on the home screen.

Reply:
Yeah, that’s a good point. I think the MOTD should be displayed on the home screen. It’s much less useful if the user has to navigate to a mint details screen to see it, especially since these are timely alerts. I updated the design so the MOTD now appears at the top of the home screen as a dismissible notification.